roles/news/templates/var_www_selfoss_config.ini.j2

```diff
@@ -1,9 +1,10 @@
 ; see https://github.com/SSilence/selfoss/wiki/Configuration for more information about the configuration parameters
 [globals]
 db_type=pgsql
-db_database={{selfoss_db_database}}
-db_username={{selfoss_db_username}}
-db_password={{selfoss_db_password}}
+db_host=127.0.0.1
+db_database="{{selfoss_db_database}}"
```

Comment:
Do you think quoting the database and username are necessary?  Just trying to focus the patch on the specific problem and not make speculative changes.